src/components/MerchantPayout/MerchantPayoutScreen.tsx

```diff
@@ -278,23 +260,10 @@ export const MerchantPayoutScreen: FunctionComponent<NavigationFocusInjectedProp
               <SecondaryButton
                 text="Cancel"
                 onPress={() => {
-                  Alert.alert(
-                    "Discard transaction?",
-                    "This will clear all scanned items",
-                    [
-                      {
-                        text: "Cancel"
-                      },
-                      {
-                        text: "Discard",
-                        onPress: () => {
-                          setMerchantCode("");
-                          resetState();
-                        },
-                        style: "destructive"
-                      }
-                    ]
-                  );
+                  showErrorAlert(new Error("discardTransaction"), () => {
+                    setMerchantCode("");
+                    resetState();
+                  });
```

Comment:
Oh can we change this to a warning alert instead? Then maybe `showWarnAlert` can take in an enum instead too just like `showConfirmationAlert`

Reply:
This has been refactored to:
```suggestion
                  showWarnAlert(WARNING_MESSAGE.DISCARD_TRANSACTION, () => {
                    setMerchantCode("");
                    resetState();
                  });
```